setup.py

```diff
@@ -5,14 +5,13 @@
     version='0.1dev',
     packages=['Colourblind8',],
     license='LICENSE.txt',
-    long_description=open('README.txt').read(),
+    long_description=open('README.md').read(),
+    long_description_content_type='text/markdown',
      author = ['Ian Flores', 'Sabrina Tse', 'Hayley Boyce '],
     install_requires=[
         "numpy",
```

Comment:
Please double check if this code is correct and working.

Reply:
This has been resolved